assets/src/blocks/Happypoint/HappypointFrontend.js

```diff
@@ -6,6 +6,8 @@ export const HappypointFrontend = ({
   mailing_list_iframe,
   iframe_url,
   id,
+  use_embed_code,
+  embed_code,
   className,
```

Comment:
Should we also fetch `happy_point_embed_code` from settings just below? Otherwise only the embed code from the sidebar is being used.

Reply:
yes indeed, was missed from my head.  PR updated :)